sources/platform/actors/running/store.md

```diff
@@ -3,129 +3,125 @@ title: Actors in store
-
-![Example run usage](./images/store/example_run_usage.png)
-
-> With this model, it's very easy to see how many platform resources each Actor run consumed, but it is quite difficult to estimate their usage beforehand. The best way to find the costs of free Actors upfront is to try out the Actor on a limited scope (for example, on a small number of pages) and evaluate the consumption. You can easily do that using our [free plan](https://apify.com/pricing).
->
-> **For more information on platform usage cost see the [usage and resources](./usage_and_resources.md) page.**
+1. [**Rental**](#rental-actors) - to continue using the Actor after the trial period, you must rent the Actor from the developer and pay a flat monthly fee in addition to the costs associated with the platform usage that the Actor generates.
+2. [**Pay per result**](#pay-per-result) - you do not pay for platform usage the Actor generates and instead just pay for the results it produces.
+3. [**Pay per usage**](#pay-per-usage) - you can run the Actor and you pay for the platform usage the Actor generates.
 
 ### Rental Actors
 
-Rental Actors are like free Actors, but with a slight difference. When you use a rental Actor, you pay a fee to its developer. This empowers the developer to dedicate more time and effort to their Actors, thus ensuring they are of the **highest quality** and receive **ongoing maintenance**.
+Rental Actors are Actors for which you have to pay a recurring fee to the developer after your trial period ends. This empowers the developer to dedicate more time and effort to their Actors, thus ensuring they are of the _highest quality_ and receive _ongoing maintenance_.
 
-![Free and rental Actors in Apify Store](./images/store/free_vs_paid_actors.png)
+![Rental Actor example](./images/store/rental-actor-example.png)
 
-Each rental Actor has a **free trial,** where the trial's length is always shown for each rental Actor.
+Each rental Actor has a _free trial_ period. The lenght of the trail is displayed in each Actor's page.
 
-![Rental Actors in Apify Store](./images/store/paid-actors-store.png)
+![Rental Actors in Apify Store](./images/store/rental-actor-trial-example.png)
 
----
+After a trial period, a flat monthly _Actor rental_ fee is automatically subtracted from your prepaid platform usage in advance for the following month. This fee goes directly to the developer and is paid on top of the platform usage generated by the Actor. You can read more about our motivation for releasing rental Actors in [this blog post](https://blog.apify.com/make-regular-passive-income-developing-web-automation-actors-b0392278d085/) from Apify's CEO Jan Čurn.
```

Comment:
`This fee goes directly to the developer`. This is actually no longer true, only 80% goes to developer. But maybe we don't have to be that specific. Let's just say `most of this fee goes directly to the developer`